src/ui/dom/ListView.ts

```diff
@@ -166,10 +188,29 @@ class ListView<T> extends SingleNodeWidget {
 			};
 		}
 
+		this._children = [];
 		this._widget.set('store', store);
 		this._collection = value;
 	}
 
+	destroy():void {
+		super.destroy();
+		ContainerMixin.prototype.destroy.call(this);
+	}
+
+	empty():void {
+		ContainerMixin.prototype.empty.call(this);
+	}
+
+	getChildIndex(child:Widget):number {
+		return ContainerMixin.prototype.getChildIndex.call(this, child);
+	}
+
+	_initialize():void {
+		super._initialize();
+		this._children = [];
```

Comment:
The list of “children” currently in the ListView could just be computed from the underlying IteratorList instead